src/makeRobot.js

```diff
@@ -38,7 +38,52 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(y = 1) {
+      if (y > 0) {
+        this.coords.y += y;
+      }
+
+      return this;
+    },
+    goBack(y = 1) {
```

Comment:
```suggestion
    },
    
    goForward(y = 1) {
      if (y > 0) {
        this.coords.y += y;
      }

      return this;
    },
    
    goBack(y = 1) {
```
Don't forget to add newlines between methods to make your code more readable